src/main/java/com/jorgepatrick/RomanCalculator.java

```diff
@@ -0,0 +1,55 @@
+package com.jorgepatrick;
+
+public class RomanCalculator {
+    public String sumRomanNumbers(final String firstAddend, final String secondAddend) {
+
+        int firstArabicNumber = 0;
+        int secondArabicNumber = 0;
+        firstArabicNumber = parseRomanToArabic(firstAddend);
```

Comment:
You can declare the `firstArabicNumber ` and `secondArabicNumber ` as final. So they would be immutable. 

```
final int firstArabicNumber = parseRomanToArabic(firstAddend);
final int secondArabicNumber = parseRomanToArabic(secondAddend);
```